sources/api/migration/migrations/v1.11.0/ecs-additional-configurations/src/main.rs

```diff
@@ -0,0 +1,32 @@
+#![deny(rust_2018_idioms)]
+
+use migration_helpers::common_migrations::AddSettingsMigration;
+use migration_helpers::{migrate, Result};
+use std::process;
+
+/// We added additional configurations for the ECS agent:
+///
+/// settings.ecs.container-stop-timeout
+/// settings.ecs.task-cleanup-wait-duration
+/// settings.ecs.metadata-service-rps
+/// settings.ecs.metadata-service-burst
+/// settings.ecs.reserved-memory
```

Comment:
nit: seems like the list in this comment just duplicates the code
```suggestion
/// We added additional configurations for the ECS agent.
```